pkgs/development/tools/phpactor/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib, stdenvNoCC, fetchFromGitHub, php, composer }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "phpactor";
+  version = "2023.01.21";
+
+  src = fetchFromGitHub {
+    owner = "phpactor";
+    repo = "phpactor";
+    rev = version;
+    hash = "sha256-jWZgBEaffjQ5wCStSEe+eIi7BJt6XAQFEjmq5wvW5V8=";
+  };
+
+  buildInputs = [
```

Comment:
should be nativeBuildInputs

Reply:
`php` probably should remain in `buildInputs`, though.